src/bootstrap/test.rs

```diff
@@ -1123,6 +1123,14 @@ default_test!(Assembly { path: "src/test/assembly", mode: "assembly", suite: "as
 
 default_test!(CBMC { path: "src/test/cbmc", mode: "rmc", suite: "cbmc" });
 
+default_test!(Firecracker { path: "src/test/firecracker", mode: "rmc", suite: "firecracker" });
+
+default_test!(Prusti { path: "src/test/prusti", mode: "rmc", suite: "prusti" });
+
+default_test!(Serial { path: "src/test/serial", mode: "rmc", suite: "serial" });
```

Comment:
I know we are not running these at the moment, but do you know how much they would take to complete?